test/common/quic/platform/quic_platform_test.cc

```diff
@@ -207,12 +207,24 @@ TEST_F(QuicPlatformTest, QuicThread) {
 
     ~AdderThread() override = default;
 
+    void waitForRun() {
+      // Wait for Run() to finish.
+      std::unique_lock<std::mutex> lk(m_);
+      cv_.wait(lk);
+    }
+
   protected:
-    void Run() override { *value_ += increment_; }
+    void Run() override {
+      std::unique_lock<std::mutex> lk(m_);
+      *value_ += increment_;
+      cv_.notify_one();
+    }
 
   private:
     int* value_;
     int increment_;
+    std::mutex m_;
```

Comment:
The convention is to use absl::Mutex which supports conditional usage as well.

Reply:
switched to absl mutex now!